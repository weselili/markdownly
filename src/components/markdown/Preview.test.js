import React from 'react';
import { shallow } from 'enzyme';
import Preview from './Preview';

describe('Preview component', () => {
  it('renders Preview', () => {
    const wrapper = shallow(<Preview markdown={'u'}/>);
    expect(wrapper).toMatchSnapshot();
  });
});
  
