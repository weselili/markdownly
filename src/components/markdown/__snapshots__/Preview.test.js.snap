// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Preview component renders Preview 1`] = `
<div
  className="markdown-body"
  dangerouslySetInnerHTML={
    Object {
      "__html": "<p>u</p>
",
    }
  }
/>
`;
